app/views/admin/portal/guide.html.haml

```diff
@@ -9,28 +9,28 @@
 
       %p Chapter organisers can create and modify sponsors and workshops only for chapters that they are permissioned. To make a member into an organiser of a chapter,
 
-      = raw @markdown.render("`Member.find(:id).add_role(:organiser, Chapter.find(:id))`")
+      = raw CommonMarker.render_html("`Member.find(:id).add_role(:organiser, Chapter.find(:id))`")
```

Comment:
can we not just use `dot_markdown(text)` in the entire guide as well? Then we would have only one place referencing common marker.